tools/scalajsenv.js

```diff
@@ -761,21 +777,24 @@ ScalaJS.TypeData.prototype.initClass = function(
 initClass(
 //!endif
     internalNameObj, isInterface, fullName,
-    ancestors, isRawJSType, parentData, isInstance, isArrayOf) {
+    ancestors, typeTag, isRawJSType, parentData, isInstance, isArrayOf) {
   const internalName = ScalaJS.propertyName(internalNameObj);
```

Comment:
`internalName` and, consequently, `internalNameObj`, are not needed anymore. They should be removed. :p